girder/web_client/src/views/widgets/ItemListWidget.js

```diff
@@ -237,11 +237,16 @@ var ItemListWidget = View.extend({
     centerSelected: function () {
         const widgetcontainer = this.$el.parents('.g-hierarchy-widget-container');
         const selected = this.$('li.g-item-list-entry.g-selected');
-
         if (widgetcontainer.length > 0 && selected.length > 0) {
-            const centerPos = (widgetcontainer.height() / 2.0) + (selected.outerHeight() / 2.0);
-            // Take the current scroll top position and add it to the position of the top of the selected item, then center it
-            const scrollPos = widgetcontainer[0].scrollTop + selected.position().top - centerPos;
+            // These items will effect the scroll position if they exists
+            const folderHeight  = $('.g-folder-list').length ? $('.g-folder-list').height() : 0;
```

Comment:
Isn't the jquery `height()` of an empty list 0?  That is, do you need the ternary on this and the next line, or can you just use `$('.g-folder-list').height()`?

Reply:
You would think so since the official return value is a number, but it will return `undefined` if the selector returns an empty array.  While I believe in most cases there will be an empty `hierarchy-breadcrumb-bar` and `g-folder-list` I can't guarantee it.  I don't like how this needs to reach up into a higher level in the structure to check for these items but it ensures the proper scrolling and highlighting.